src/org/labkey/test/components/ui/ontology/ConceptPickerDialog.java

```diff
@@ -116,6 +115,12 @@ public ConceptPickerDialog selectOntology(String ontology)
         return select.getOptions();
     }
 
+    public ConceptPickerDialog waitForActiveTreeNode()
```

Comment:
getVisibleActiveNode() should already wait for the node, if all you need is the wait/sync